src/components/digitalSpecimen/DigitalSpecimen.tsx

```diff
@@ -99,6 +100,19 @@ const DigitalSpecimen = () => {
         }
     });
 
+    /**
+     * Function to set the annotation target state
+     * @param annotationTargetType The type of the annotation target, either class or term
+     * @param jsonPath The JSON path that targets the class or term
+     */
+    const SetAnnotationTarget = (annotationTargetType: 'class' | 'term', jsonPath: string) => {
```

Comment:
A lot of this seems duplicate from the media.tsx file, do you think you can reuse some code?

Reply:
Yeah my initial idea was to include as an utility function, only I am not allowed to use hooks out of React components, hence I chose to do it this way. I could include it at an ever higher level and pass it as props, but its irrelevant for a lot of child components. For now this seemed the best way, but can probably be optimized somehow in the future.